WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -236,3 +237,36 @@ $(function() {
 Math.trunc = Math.trunc || function(x) {
   return x < 0 ? Math.ceil(x) : Math.floor(x);
 };
+
+// Setting up bootstrap-table
+$(function() {
+  // Hide loading box
+  $('table[data-toggle="table"]').bootstrapTable('hideLoading');
+
+  // Triggered when a sort arrow is clicked but before a table is sorted
+  $('table').on('sort.bs.table', function(e, name, order) {
+    // The table column that we are sorting by
+    var field = $('table').find('th[data-field="' + name + '"] .sortable');
+    // If it's not the field we are currently sorting by
+    if(!field.is('.asc, .desc')) {
+      // Change the sort order that's set in data-order ('asc' by default)
+      var options = $(this).bootstrapTable('getOptions');
+      options.sortOrder = options.columns[0].find(function(option) { return option.field == name; }).order;
+      // Now the table will be sorted using the order that we set
+    }
+  });
+
+  // Set values of checkboxes in a table to corresponding rows ids
+  var initCheckboxesValues = function(table) {
+    var selectRowCeckbox = table.find('tr td input[type="checkbox"]');
```

Comment:
Maybe change this selector to use `td:first-child` instead?

We have a convention of always naming variables that are jquery objects with a $ first. So could you change this from `table` to `$table`, and `selectRowCeckbox` to `$selectRowCheckbox` (lol spelling)?